inc/environment_indicator/namespace.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Altis\Cloud\Environment_Indicator;
+
+use function Altis\get_environment_name;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'admin_menu', __NAMESPACE__ . '\\add_menu_page' );
+}
+
+function add_menu_page() {
+	\add_menu_page(
```

Comment:
I was after adding this to the admin bar rather than admin menu, sorry it wasn't clearer